src/Definition/Source/SourceCache.php

```diff
@@ -23,21 +23,39 @@ class SourceCache implements DefinitionSource, MutableDefinitionSource
      */
     private $cachedSource;
 
+    /**
+     * @var null|string
+     */
+    private static $cacheKeyPrefix;
+
     public function __construct(DefinitionSource $cachedSource)
     {
         $this->cachedSource = $cachedSource;
     }
 
+    /**
+     * @return string
+     */
+    public static function getCacheKey(): string
+    {
+        if(self::$cacheKeyPrefix === null)
+        {
+            self::$cacheKeyPrefix = str_replace([':', DIRECTORY_SEPARATOR], '', __DIR__);
+        }
+
+        return self::$cacheKeyPrefix . '_' . self::CACHE_KEY;
```

Comment:
could you list here in a comment an exemplary cache prefix, what's generated from the code, please?

Reply:
homeleadeabkwww.leadersnet.atvendorovosphp-disrcDefinitionSource

Reply:
`homedsdafxxwww.example.comvendorovosphp-disrcDefinitionSource`